HISTORY.md

```diff
@@ -4,13 +4,16 @@ Release History
 dev
 ---
 
--   \[Short description of non-trivial change.\]
-
 2.26.0 (2021-07-13)
 -------------------
 
 **Improvements**
 
+- Added a `requests.exceptions.JSONDecodeError` to decrease inconsistencies
+  in the library. This gets raised in the `response.json()` method, and is
+  backwards compatible as it inherits from previously thrown exceptions.
+  Can be caught from `requests.exceptions.RequestException` as well.
+
```

Comment:
2.26.0 was released. This can not be added to those notes as this was not released with 2.26.0 please move it above the `2.26.0 (2021-07-13)` line.